networking/external_dns_operator/nw-installing-external-dns-operator.adoc

```diff
@@ -0,0 +1,34 @@
+[id="nw-installing-external-dns-operator_{context}"]
+= Installing the External DNS Operator
+
+You can install External DNS Operator using the Red Hat OpenShift Container Platform Operator Hub.
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+* You have created the `operand` namespace.
+* You have created the `external-dns` namespace.
```

Comment:
From my understanding, the operand namespace is just `external-dns`, so the two statements can be replaced by one: You have created the operand `external-dns` namespace.

Reply:
Addressed